File: stow_packages/awesome/.config/awesome/visual/themes/basic/colorschemes/gruvbox/colors.lua
require("visual.colorschemes")

local colors = GRUVBOX

other_colors = {
  light = colors.white,
  gray = colors.gray1,
  dark = colors.black,
  too_dark = colors.black0,

  font = colors.white,
  bg = colors.black,
  border = colors.black,
}

---{{{ Widget Colors
app_launcher_colors = {
  fg = other_colors.dark,
  fg_icon = colors.black,
  bg = colors.red2,
  border = colors.red1,
}

battery_colors = {
  fg = colors.black,
  fg_icon = colors.black,
  bg = colors.blue2,
  bg_icon = colors.blue1,
  border = colors.blue1,
}

time_colors = {
  fg = other_colors.dark,
  fg_icon = colors.black,
  bg = colors.aqua2,
  bg_icon = colors.aqua1,
  border = colors.aqua1,
}

date_colors = {
  fg = other_colors.dark,
  fg_icon = colors.black,
  bg = colors.purple2,
  bg_icon = colors.purple1,
  border = colors.purple1,
}

tray_colors = {
  fg = other_colors.dark,
  fg_icon = colors.black,
  bg = colors.orange2,
  border = colors.orange1,
}
---}}}

taglist_colors = {
  fg = colors.gray1,
  bg = colors.green2,
  bg_focus = colors.green1,
  bg_urgent = colors.black,
  fg_focus = colors.black,
  fg_occupied = colors.black,
  fg_urgent = colors.red1,
  border = colors.green1,
  hover = colors.gray2,
}

tasklist_colors = {
  fg = colors.orange1,
  fg_focus = colors.white,
  fg_minimize = colors.red2,
  bg = colors.black,
  bg_focus = colors.orange2,
  bg_minimize = colors.red1,
  border = colors.black,
}

titlebar_colors = {
  border = colors.black,
  border_focus = colors.green2,
}

bar_colors = {
  bg = colors.black,
  border = colors.black,
}

File: stow_packages/qtile/.config/qtile/themes/colorschemes/gruvbox/colors.py
from themes.core_colorschemes import gruvbox

c = gruvbox

other_colors = {
    "light": c["white"],
    "gray": c["gray1"],
    "dark": c["black"],
    "too_dark": c["black0"],
    "font": c["white"],
    "bg": c["black"],
    "border": c["black"],
}

# Widget Colors
app_launcher_colors = {
    "fg": c["black"],
    "bg": c["green2"],
}

tray_colors = {
    "fg": ["black"],
    "fg_icon": c["black"],
    "bg": c["purple2"],
}

windowname_colors = {
    "fg": c["white"],
    "bg": c["black"],
}

windowcount_colors = {
    "fg": c["black"],
    "bg": c["blue2"],
}

groupbox_colors = {
    "bg": c["orange2"],
    "bg_focus": c["orange1"],
    "bg_urgent": c["orange2"],
    "fg_active": c["black"],
    "fg_inactive": c["orange1"],
    "fg_focus": c["black"],
    "fg_urgent": c["red1"],
}

layout_colors = {
    "bg": c["blue2"],
}

volume_colors = {
    "fg": c["black"],
    "bg": c["red2"],
}

brightness_colors = {
    "fg": c["black"],
    "bg": c["purple2"],
}

network_colors = {
    "fg": c["black"],
    "bg": c["green2"],
}

widgetbox_colors = {
    "fg": c["red1"],
    "bg": c["red1"],
}

battery_colors = {
    "fg": c["black"],
    "fg_icon": c["black"],
    "fg_low": c["red1"],
    "bg": c["blue2"],
    "bg_icon": c["blue1"],
    "bg_low": c["red2"],
}

time_colors = {
    "fg": c["black"],
    "fg_icon": c["black"],
    "bg": c["aqua2"],
    "bg_icon": c["aqua1"],
}

date_colors = {
    "fg": c["black"],
    "fg_icon": c["black"],
    "bg": c["yellow2"],
    "bg_icon": c["yellow1"],
}

client_colors = {
    "border": c["black"],
    "border_focus": c["orange2"],
    "border_floating": c["green2"],
}

bar_colors = {
    "bg": c["black"],
    "border": c["black"],
}
